src/plugins/data/config.ts

```diff
@@ -48,6 +48,9 @@ export const configSchema = schema.object({
         enabled: schema.boolean({ defaultValue: false }),
       }),
     }),
+    usageTelemetry: schema.object({
+      enabled: schema.boolean({ defaultValue: false }),
```

Comment:
Should this be true since currently it's always on? I can see the argument though about defaulting to false. However, since we are going to `2.0.0` now's the time to change the default behavior.

Reply:
This seems like that the same comment as above. Yes. it is default behavior change, but not feature break change, customer still can turn it on if they needed. we will keep it as false for now and document it when release.

Reply:
Gotcha. I would still factor as a breaking change because it changes the default behavior and the end user will now need to add this config. But I do not have the insist on how useful the telemetry out of the box experience has been and from one customer's perspective has been detrimental for performance then I'd rather change the default behavior to have people opt in to this telemetry. But we should ensure that the documentation website is very explicit in this notice that if a community has been expecting the default behavior to include this telemetry out of the box that they will need to explicitly set it.

Reply:
Not sure why you got double messages from me last night. Apologies !

Reply:
Thanks. We will documentation this detail information in the community for the default behavior change and provide the guide for opt in/out for this.